style/aboutwikipedia.less

```diff
@@ -43,8 +43,8 @@
 
 		.title {
 			font-weight: bold;
-			font-size: 22px;
-			line-height: 26px;
+			font-size: 20px;
+			line-height: 23.6px;
```

Comment:
Assuming that .6 is not a typo

Reply:
nope, 

_unfortunately this is not on the ticket but the value is given through direct message_